lib/install.ps1

```diff
@@ -582,6 +582,8 @@ function dl_urls($app, $version, $manifest, $bucket, $architecture, $dir, $use_c
         $extract_fn = $null
         if ($manifest.innosetup) {
             $extract_fn = 'Expand-InnoArchive'
+        } elseif($manifest.innoexsetup) {
+            $extract_fn = 'Expand-InnoExArchive'
         } elseif($fname -match '\.zip$') {
             # Use 7zip when available (more fast)
             if (((get_config 7ZIPEXTRACT_USE_EXTERNAL) -and (Test-CommandAvailable 7z)) -or (Test-HelperInstalled -Helper 7zip)) {
```

Comment:
You need to use something like this:

Best possible add simple switch parameter (`-UseInnoExtract` or just `-InnoExtract`) to expand-innoarchive function

Reply:
I take it back. Sorry.

Only use this

`if ($manifest.innosetup) { if (get_config 'INNOSETUP_USE_INNOEXTRACT' $false) { $depends += 'innoextract' } } else { $depends += 'innounp' }`

And then inside function Expand-InnoArchive use same logic:

```
if get_config ''... {
 $Stratus = INvoke-external Innoextract
} else  {
$status = Invoke-External innounp }
```

Reply:
Hi, to add the switch parameter/option, do I edit `scoop-install.ps1` and `scoop-update.ps1` in scoop\libexec?

Reply:
@Ash258 May I know how exactly do I add a switch parameter? Thanks.